neqo-transport/src/packet.rs

```diff
@@ -37,6 +37,7 @@ const SAMPLE_SIZE: usize = 16;
 const SAMPLE_OFFSET: usize = 4;
 
 pub type PacketNumber = u64;
+type Version = u32;
```

Comment:
Do we still use this?  I mean, other than the `as_u32` function and `TryFrom` implementation below, which is clearly something we could change to just be `u32`

Reply:
```
        // Generic long header.
        let version = Version::try_from(Self::opt(decoder.decode_uint(4))?).unwrap();
```
Should we just eliminate the type and use `u32` instead?

Reply:
That is essentially what I'm suggesting.